packages/dhis-api/src/__tests__/DhisApi/testGetEventAnalytics.js

```diff
@@ -59,14 +59,35 @@ export const testGetEventAnalytics = () => {
 
   it('translates codes to ids in the provided query', async () => {
     await dhisApi.getEventAnalytics(QUERY.originalInput);
-    expect(BuildAnalyticsQuery.buildEventAnalyticsQuery).toHaveBeenCalledOnceWith(
+    return expect(BuildAnalyticsQuery.buildEventAnalyticsQuery).toHaveBeenCalledOnceWith(
```

Comment:
Ooh, good catch

Reply:
I'm not sure how but I think it works either way... 
I raised https://github.com/beyondessential/tupaia-backlog/issues/1599 to catch this issue, but even without the return it was working... magic. I added it in for safety.

Reply:
Surprising - I wonder if the whole test suite ended before the async function had completed whether the failure would get missed. Definitely safer the way you've got it.